pydis_site/apps/content/resources/guides/pydis-guides/contributing/working-with-git/pycharm.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Working with Git in PyCharm
+description: Basic workflow when using git in PyCharm.
+---
+
+This is the basic workflow when working with Git with PyCharm. For the CLI version of the guide, [**click here**](../cli).
+The following will use the [Sir-Lancebot](https://github.com/python-discord/sir-lancebot/) repository as an example, but the steps are the same for all other repositories.
+
+> **Note:** This is a guide only meant to get you started with git. For in-depth resources, check the [**Working with Git**](wiki:/contributing/working-with-git/) page.
+
+#### Only done once per project
+* [Adding the Upstream Remote](adding-the-upstream-remote)
+
+#### Done for each feature
+* [Creating a New Branch](creating-a-new-branch)
+* [Committing Changes](committing-changes)
+* [Pushing Changes](pushing-changes)
+
+#### Done whenever needed
+* [Pulling Changes](pulling-changes)
+
+---
+
+## Adding the Upstream Remote
+> Adding a *remote* to the main GitHub repository you forked off will allow you to later update your fork with changes from the main repository.
+
+> Generally, a *remote* designates a repository that is on GitHub or another external location rather than on your computer. The `origin` remote will refer to your fork on GitHub. The `upstream` remote will refer to the main repository on GitHub.
+
+1. In the menu bar, navigate to `Git` -> `Remotes...`.  
+![PyCharm Remotes](/static/images/content/contributing/pycharm_remotes.webp)
+2. In the popup menu, click the `+` icon, set `upstream` as the name, set the URL as the URL for the main repository on GitHub.  
+![PyCharm Upstream Project](/static/images/content/contributing/pycharm_upstream.webp)
+3. Click `OK`.
+
+---
+
+## Creating a New Branch
+> You will be committing your changes to a new branch rather than to `main`. Using branches allows you to work on multiple pull requests at the same time without conflicts.
+
+> You can name your branch whatever you want, but it's recommended to name it something succint and relevant to the changes you will be making. 
+
+> Before making new branches, be sure to checkout the `main` branch and ensure it's up to date.
+
+1. In the bottom right corner, click on `main` and then click `New Branch`.    
+![PyCharm New Branch](/static/images/content/contributing/pycharm_branch.webp)
```

Comment:
This image is especially blurry/low res and appears to be scaled up. It seems like that's cause the original image was already pretty bad and the compression artefacts you've added made aggravated it.

Reply:
This is solved with the move back to PNG.